MEME/MEME/Model/MyReview/MyReviewViewController.swift

```diff
@@ -408,24 +457,50 @@ class MyReviewViewController: UIViewController, UICollectionViewDelegate, UIColl
         menuView.addArrangedSubview(deleteButton)
         editButton.heightAnchor.constraint(equalToConstant: 50).isActive = true
         deleteButton.heightAnchor.constraint(equalToConstant: 50).isActive = true
-
+        
         menuView.translatesAutoresizingMaskIntoConstraints = false
         let cellBottomRight = CGPoint(x: cell.bounds.width, y: cell.bounds.height)
         let cellPositionInSuperview = cell.convert(cellBottomRight, to: self.view)
-
+        
         NSLayoutConstraint.activate([
             menuView.rightAnchor.constraint(equalTo: self.view.leftAnchor, constant: cellPositionInSuperview.x - 10),
             menuView.topAnchor.constraint(equalTo: self.view.topAnchor, constant: cellPositionInSuperview.y - 30),
             menuView.widthAnchor.constraint(equalToConstant: 85),
             menuView.heightAnchor.constraint(equalToConstant: 56)
         ])
-
+        
     }
     
     @objc func moveToReviewEditVC() {
-//        let reviewEditVC = ReviewEditViewController()
-//           reviewEditVC.reviewContent = "리뷰 내용" // 수정할 리뷰의 내용
-        navigationController?.pushViewController(WriteReviewViewController(), animated: true)
+        let writeReviewVC = WriteReviewViewController()
+        
+        DetailReviewManager.shared.getDetailReview(reviewId: 1, artistNickName: "String", makeupName: "String", star: 3, comment: "String", reviewImgDtoList: [DetailReviewImage]()) { [weak self] result in
+            switch result {
+            case .success(let response):
+                if let responseData = response.data {
+                    DispatchQueue.main.async {
+                        writeReviewVC.reviewLabel.text = responseData.makeupName
+                        writeReviewVC.updateReviewLabel(artistName: responseData.artistNickName, makeupName: responseData.makeupName)
+                        
+                        if let firstImageUrlString = responseData.reviewImgDtoList.first?.reviewImgSrc, let imageUrl = URL(string: firstImageUrlString) {
```

Comment:
```suggestion
                        if 
                            let firstImageUrlString = responseData.reviewImgDtoList.first?.reviewImgSrc,
                            let imageUrl = URL(string: firstImageUrlString)
                         {
```
과 같이 작성하면 가독성이 더 좋아질 것 같습니다 ㅎㅎ